coordinator/src/main/scala/filodb.coordinator/DatasetCoordinatorActor.scala

```diff
@@ -132,6 +133,7 @@ private[filodb] class DatasetCoordinatorActor(projection: RichProjection,
   var flushesStarted = 0
   var flushesSucceeded = 0
   var flushesFailed = 0
+  var rowsIngested = 0L
```

Comment:
private[filodb] case class DatasetStats(flushesStarted: Int, flushesSucceeded: Int, flushesFailed: Int, rowsIngested: Long)`

      private class DatasetCoordinatorActor ... {
        private var stats = DatasetStats.empty //initialize in the apply
     }